test/e2e/provisioning/all_e2e_test.go

```diff
@@ -166,6 +167,24 @@ func TestHetznerProvisioningE2E(t *testing.T) {
 	runScenarios(t, excludeSelector, params, HZManifest, fmt.Sprintf("hz-%s", *testRunIdentifier))
 }
 
+// TestLinodeProvisioning - a test suite that exercises Linode provider
+// by requesting nodes with different combination of container runtime type, container runtime version and the OS flavour.
+//
+// note that tests require  a valid API Token that is read from the LINODE_E2E_TEST_TOKEN environmental variable.
+func TestLinodeProvisioningE2E(t *testing.T) {
+	t.Parallel()
+
+	// test data
+	linodeToken := os.Getenv("LINODE_E2E_TESTS_TOKEN")
```

Comment:
Could you add an `excludeSelector` so only Ubuntu is tested as that is the only supported OS?

https://github.com/kubermatic/machine-controller/blob/3f15ca3bcbb4c18c408be72c59f2bcd80483d81a/test/e2e/provisioning/all_e2e_test.go#L54

Reply:
Will do.

Reply:
updated in [575403a](https://github.com/kubermatic/machine-controller/commit/575403ae478681a8e3395eeabbb8588c522309a5)